.github/workflows/testing.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           - os: macos-latest
             # numpy triggers: RuntimeError: Polyfit sanity test emitted a
             # warning
-            python_version: "pypy3"
+            python_version: "pypy-3.8"
```

Comment:
would it be possible to explicitly set one of those 2 pypy config to use pypy-3.7?

Reply:
actually I was confused by the way the matrix is defined negatively. We need to add `"pypy-3.7"` to the list of Python version and then exclude the all the macos / windows + pypy combos that still fail.